examples_tests/09.ColorSpaceTest/main.cpp

```diff
@@ -333,9 +300,7 @@ int main()
 			beginInfo.clearValues = &clear;
 
 			commandBuffer->beginRenderPass(&beginInfo, nbl::asset::ESC_INLINE);
-			commandBuffer->bindGraphicsPipeline(gpuGraphicsPipeline.get());
-			commandBuffer->bindDescriptorSets(asset::EPBP_GRAPHICS, currentGpuRenderpassIndependentPipeline->getLayout(), 3u, 1u, &gpuSamplerDescriptorSet3.get(), nullptr);
-			commandBuffer->drawMeshBuffer(gpuMeshBuffer.get());
+			ext::FullScreenTriangle::recordDrawCalls(commandBuffer.get(), gpuGraphicsPipeline.get(), &gpuSamplerDescriptorSet3.get());
```

Comment:
expect user will bind graphcis pipeline and descriptor set by themselves.

Reply:
in that case is this helper function even needed? There will be only one `draw` invocation

Reply:
OH OKAY, there won't https://github.com/Devsh-Graphics-Programming/Nabla/pull/172#discussion_r688927109, and forgot about vertex index bindings at all XD

Reply:
corrected